build.bat

```diff
@@ -269,6 +270,7 @@ set rime_cmake_flags=%common_cmake_flags%^
  -DBUILD_TEST=%build_test%^
  -DENABLE_LOGGING=%enable_logging%^
  -DCMAKE_CONFIGURATION_TYPES="%build_config%"^
+ -DCMAKE_BUILD_TYPE:STRING="%build_config%"^

```

Comment:
I don't believe these 2 options can share the same value.

Reply:
ninja used DCMAKE_BUILD_TYPE 
msvc used  CMAKE_CONFIGURATION_TYPES

ref: https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#build-configurations